### DISCUSSION TITLE
Accessing request body in filters

### DISCUSSION THREAD

**Discussion by hertg** [playframework/playframework#12656] [Show and tell]
I recently had to figure out how to access the request body from within Play filters. I found many questions about it online, but no answer that satisfied my requirements. So I came up with a solution that is based on the Play internal `CsrfFilter`, but with some small differences. For anyone interested, I wrote about it on my blog.

https://hertig.blog/posts/accessing-request-body-in-play-framework-filters/

And I also created a public repository, with a specific implementation for parsing bearer tokens from the incoming request, based on RFC 6750.

https://github.com/hertg/play-rfc6750

**Comment by mkurz** [comment]
Thanks for sharing!
